tests/pulp_test.py

```diff
@@ -22,17 +22,38 @@ def test_pulp_service(host):
     assert pulp.is_running
     assert pulp.is_enabled
 
+def test_pulp_api_service(host):
+    pulp_api = host.service("pulp-api")
+    assert pulp_api.is_running
+    assert pulp_api.is_enabled
+
+def test_pulp_content_service(host):
+    pulp_content = host.service("pulp-content")
+    assert pulp_content.is_running
+    assert pulp_content.is_enabled
+
+def test_pulp_worker_services(host):
+    for i in range(1, 3):
+        pulp_worker = host.service(f"pulp-worker@{i}")
```

Comment:
We currently do not deploy the worker service as a template, so this test fails.